Common/yaml/category.yaml

```diff
@@ -0,0 +1,353 @@
+openapi: 3.0.3
+info:
+  title: Smart Contact Manager - OpenAPI 3.0
+  description: 
+              This Project was about creatig smart contact manager.
+              Which can be specifically used for business model.This is web 
+              application which can be used as a simple plugging to any 
+              exixting project.|-
```

Comment:
Remove the unnecesary pipe and hyphen symbol

Reply:
I have fixed it. Please review again.